libs/api/windows_platform.cpp

```diff
@@ -18,9 +18,11 @@ parse_maps_section_windows(
     std::vector<EbpfMapDescriptor>& verifier_map_descriptors,
     const char* data,
     size_t size,
+    int map_count,
     const struct ebpf_platform_t*,
     ebpf_verifier_options_t)
 {
+    UNREFERENCED_PARAMETER(map_count);
```

Comment:
map_count is being used below. so we can remove this ?

Reply:
This change is from PR #765 that this is based on top of.  Shankar comment on this over there.  This file will disappear from this PR once 765 is merged and this is rebased.